docs/content/en/docs/Customizing/configure_grub.md

```diff
@@ -94,13 +96,13 @@ For instance use the following command to reboot to recovery system only once:
 ```
 
 {{% alert title="Note" %}}
-The examples below make of the `COS_OEM` device, however it could use any device
-detected by GRUB2 that includes the file `/grubenv`. First match wins.
+The examples below make of the `COS_STATE` device, only files in the state
+partition will be used when booting.
```

Comment:
I am not sure about this note, in fact `/grubenv` is stored within the OEM paratition. In fact this should be only file stored in a RW partition, the other grub files are in state and supposed to be RO. Probably the note should simply state that `/oem/grubenv` is the only file under a RW partition, others are in state.

""

Reply:
Yep, good point! Added oem partition in the note.